### ISSUE TITLE
Continuation of development of Learning Mode?

### ISSUE THREAD

**Issue by MatthijsBlom** [exercism/haskell#1132]
@petertseng

Due to #1127, any PR I would submit would be closed with a suggestion to discuss the matter on the forum first. However, you aren't on the forum, so I'm asking here.

Are PRs working on Learning Mode still welcome? If yes, I'll continue writing them and try to have them test-driven by forum members.

**Comment by petertseng** [comment]
I'm still willing to review such pull requests, though due to other items such as travel my time is limited until January. So you've got some time.

**Comment by kytrinyx** [comment]
@MatthijsBlom this is a great question.

When we originally started working on this my primary goal was to figure out what we can learn about developing Learning Mode for a track. The process was very successful in that we learned a ton about why this is so difficult to get right.

I've spent a lot of time reflecting on it, and came to the somewhat painful conclusion that the core team doesn't have the capacity to take this on at the moment, or in the near future.

If you are motivated to work on this, that's much appreciated! And if @petertseng is willing to jump in and review, that's wonderful.

Our sense is that having a rough draft that can be refined later is a great step in the right direction.

Our overall guideline here would be: don't aim for perfection. Merge optimistically, and refine iteratively.
